src/dlgAboutDialog.cpp

```diff
@@ -183,6 +254,22 @@ void dlgAboutDialog::setAboutTab(const QString& htmlHead) const
     // clang-format on
 }
 
+QString dlgAboutDialog::createMakerHTML(const aboutMaker maker) const
+{
+    QString coloredText("<span style=\"color:#%1;\">%2</span>");
+    QStringList contactDetails;
+    if (!maker.discord.isEmpty()) {contactDetails.append(coloredText.arg("7289DA", maker.discord));}
+    if (!maker.github.isEmpty()) {contactDetails.append(coloredText.arg("40b040", maker.github));}
```

Comment:
Um, the color string here and in three other place (1 above, 2 below) also should have `QStringLiteral(....)` wrappings... :wink:

Reply:
Committed